### ISSUE TITLE
[Mod Request]: Cooking for Blockheads

### ISSUE THREAD

**Issue by pollymonosumo93** [FTBTeam/FTB-University#128]
* Mod Name: Cooking for Blockheads
* Mod Dependencies: None
* Mod License:MIT License 
* Mod Link:https://www.curseforge.com/minecraft/mc-mods/cooking-for-blockheads
* Why you want it: this would make the more involved cooking recipes easier to craft

**Comment by NuclearDogs-1** [comment]
Make sure you select required dependencies for it, it does not actually require any of the mod dependencies stated above.

**Comment by SissyFemBoi** [comment]
I thought Cooking for Blockheads needed those mods because they are in the mod Dependencies tap, so was I incorrect that Cooking for Blockheads needed those mods?

**Comment by pollymonosumo93** [comment]
I thought Cooking for Blockheads needed those mods because they are in the mod Dependencies tap, so was I incorrect that Cooking for Blockheads needed those mods?

**Comment by NuclearDogs-1** [comment]
Indeed, it can be confusing, because optional dependencies are mods that are supported by the mod you are looking at. For example, with enderio it supports AE2 with its ME conduits and thus AE2 is an optional dependency, you just have to double check if the mod cannot run without the dependency. And put any of those mods in the dependencies option.

**Comment by pollymonosumo93** [comment]
would you say I should edit it and remove the mods in the Dependencies and write none instead
